README.md

```diff
@@ -26,20 +26,26 @@ That way, common web components, such as payment/donation buttons, can easily be
 To get your Erdstall app started, there are a few steps you need to take.
 
 **Importing Erdstall**&emsp;<!--
--->First, import the SDK's main source directory:
+-->The Erdstall SDK is not published as a node package yet.
+It has to be manually cloned and locally registered as a node module like this:
+```sh
+	git clone https://github.com/perun-network/erdstall-ts-sdk.git erdstall
+  cd erdstall
+  yarn devpub # builds the module into dist/ and creates publishing package.json
+	yarn link # creates link in yarn cache at package @polycrypt/erdstall
```

Comment:
One has to enter the `dist` folder, otherwise it won't work 100% of the time.

```bash
cd erdstall
yarn devpub
cd dist
yarn link
```

Reply:
oh yeah, sorry